src/main/java/PacMan.java

```diff
@@ -0,0 +1,120 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class PacMan {

```

Comment:
I like how you named your chatbot PacMan! Putting a figure behind the chatbot gives it more character.